### ISSUE TITLE
Keeping Options on Links

### ISSUE THREAD

**Issue by erayerdin** [mshang/syntree#20]
It is great to have links of any tree. However, it would be better if we also kept the options in the link too. Some might prefer non-lined terminals or another might have a special font-size, height or weight etc.

The only annoying thing would be extremely long links, but there is nothing to do since it's not server side to keep short links and redirect to the tree.

**Comment by mshang** [comment]
Agreed. I would merge this, but I have no plans to implement it myself.
